config/rspack/serverRspackConfig.js

```
@@ -0,0 +1,122 @@
+      });
+      if (cssLoader && cssLoader.options && cssLoader.options.modules) {
+        // Preserve existing modules config but add exportOnlyLocals for SSR
+        cssLoader.options.modules = {
+          ...cssLoader.options.modules,
+          exportOnlyLocals: true,
+        };
+      }
+
+      // Skip writing image files during SSR by setting emitFile to false
+    } else if (rule.use && (rule.use.loader === 'url-loader' || rule.use.loader === 'file-loader')) {
+      rule.use.options.emitFile = false;
```

ESLint

[error] Assignment to property of function parameter 'rule'.

(no-param-reassign)